### ISSUE TITLE
Add vintages of truth data to visualization

### ISSUE THREAD

**Issue by nickreich** [reichlab/covid19-forecast-hub#1666]
similar to how we have at flusightnetwork.io - can we use the covidcast API to extract and display the vintages of CSSE data "as of" each Monday?

**Comment by ydhuang28** [comment]
After some investigation, I believe issues #1850 and #1851 should be merged into this one. Since JHU's covidcast package already provides a Python client with [an easy to use function](https://cmu-delphi.github.io/covidcast/covidcast-py/html/signals.html) for getting both the latest truth data and truth data with lags (vintage), I think moving entirely over to covidcast is beneficial since the amount of data parsing on our end can be significantly reduced.

Reference:
`covidcast.signal()` can provide us with both county-level and state-level truth data in easy-to-manipulate Pandas dataframes. We can then modify the dataframes to our visualization's format and aggregate for each epiweek (since the API provide values for every day).

**Comment by nickreich** [comment]
In the spirit of moving everything to covidData, should we be using that package instead of covidcast to do this parsing? FWIW - there have been issues with the JHU backfill data from covidcast.

**Comment by ydhuang28** [comment]
Yep! My last comment was made before the truth data discussions.

**Comment by ydhuang28** [comment]
Will be incorporated into v2.0.
